spec/game_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+describe "game" do
+
+  describe "Guess!" do
+    before :each do
+      dictionary = Dictionary.new %w(apple banana cheese danish egg fry)
+      @game = Cinch::Plugins::Game.new(dictionary,dictionary, solution: "banana")
+    end
```

Comment:
I debated having this a `before :all` but `each` seems safer :smile: , since it restores game and dict to a well-known state every time.